packages/react-components/react-tags/stories/Tag/TagDefault.stories.tsx

```diff
@@ -1,20 +1,148 @@
 import * as React from 'react';
 import { Tag, TagProps } from '@fluentui/react-tags';
-import { Calendar3Day28Regular } from '@fluentui/react-icons';
+import { Calendar3Day20Regular } from '@fluentui/react-icons';
+import { Avatar } from '@fluentui/react-components';
 
+// TODO I added many examples here for easier implementation. This story will be simplified to keep only the default example
 export const Default = (props: Partial<TagProps>) => (
-  <Tag
-    avatar={{
-      name: 'Katri Athokas',
-      badge: { status: 'busy' },
-      image: {
-        src: 'https://fabricweb.azureedge.net/fabric-website/assets/images/avatar/KatriAthokas.jpg',
-      },
-    }}
-    icon={<Calendar3Day28Regular />}
-    primaryText="Primary text"
-    secondaryText="Secondary text"
-    dismissable={true}
-    {...props}
-  />
+  <div style={{ display: 'flex', columnGap: 28 }}>
```

Comment:
Nit: use `makeStyles` for the examples as that's best practice.

Reply:
I updated all stories to use makeStyles except for the default stories. The default stories now contain may tags just so I have an easier time working on them. I left a TODO there to eventually have only one tag/tagButton in the default stories.